### ISSUE TITLE
do you have any solution about react-native-web Touchable's issue that you create? 

### ISSUE THREAD

**Issue by huangxing1995** [bcpugh/DjangoRibbit#1]
sorry, I don't know how to contact you , so I create a new issue for your project......

```On mobile Safari, when I fire a touchable opacity normally I get a and a touchstart and touchend event. However, if the next view (triggered by the onPress) has a touchable in the same place on the screen it immediately triggers an additional: mouseover, mousemove, mousedown, mouse up, and (2x)click events, causing the next touchable to be triggered as well.

E.g., a touchable opacity A that filled the entire screen (onPress navigates to the next view), touchable opacity B that only filled the right hand side of the screen. Clicking on the left side of touchable opacity A behaves as expected, clicking on the right side of touchable opacity A triggers the additional events, causing both touchable opacity A and subsequently B (almost immediately) to be triggered.

I'm afraid I might be in a bit over my head here-- please direct me on how I can help diagnose more specifically.

RNW: 0.9.6
RN: 0.57.4
Browser: Safari (iOS12, iPhone 7)```

for this problem, do you have any solution, can you teach me?

**Comment by bcpugh** [comment]
Sorry, as far as I know this issue is still open. In the short term we have focused on becoming more “strategic” about our UI placement and have since shifted our focus more to Native.

As an aside, I found this tip to really help reduce the buginess of rendering / firing touchables. We did not previously include this in our root html file and this improved things quite a bit. Good luck!



https://github.com/necolas/react-native-web/blob/master/docs/guides/client-side-rendering.md
